src/lib.rs

```diff
@@ -49,6 +49,9 @@ impl TreeState {
         }
     }
 
+    /// Open a tree node.
+    /// Return `true` if the node was closed and has been opened.
+    /// Return `false` if the node was already open.
```

Comment:
```suggestion
    /// Returns `true` if the node was closed and has been opened.
    /// Returns `false` if the node was already open.
```

Reply:
I have no strong preference, because the intent is clear either way. So it's mostly a stylistic thing.